gravitee-apim-console-webui/src/management/api-score/rulesets/api-score-rulesets.component.html

```diff
@@ -28,21 +28,27 @@ <h1 class="title">Rulesets</h1>
 
   <div class="separator-hr"></div>
 
-  <mat-card-content>
-    <mat-expansion-panel>
-      <mat-expansion-panel-header>
-        <mat-panel-title>
-          Default Ruleset provided by Gravitee
-          <span class="gio-badge-neutral">5 rules</span>
-          <span class="gio-badge-accent">Gravitee</span>
-        </mat-panel-title>
-        <span class="body-2 subtitle">This is the Gravitee’s default ruleset governing API.</span>
-      </mat-expansion-panel-header>
-
-      <div class="separator-hr"></div>
-
-      <h4>Raw file preview</h4>
-      <pre class="file-preview">{{ yamlFileExample }}</pre>
-    </mat-expansion-panel>
-  </mat-card-content>
+  @if (rulesets$ | async; as rulesets) {
+    <div class="rulesets">
+      @for (ruleset of rulesets.data; track ruleset.id) {
+        <mat-card-content>
+          <mat-expansion-panel>
```

Comment:
we should iterate and create multiple <mat-expansion-panel>, keep one <mat-card-content>..

Reply:
I wasn't sure what you meant here, but I changed this part to create one _mat-card-content_ component and multiple _mat-expansion-panels_ instead of multiple _mat-card-contents_ ;)

Reply:
yes, github cut some of the content...